frontend/ui/widget/frontlightwidget.lua

```diff
@@ -19,6 +19,7 @@ local UIManager = require("ui/uimanager")
 local VerticalGroup = require("ui/widget/verticalgroup")
 local VerticalSpan = require("ui/widget/verticalspan")
 local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local NaturalLight = require("ui/widget/naturallightwidget")
```

Comment:
Please stick it in alphabetically. See https://github.com/koreader/koreader-base/wiki/Coding-style#order-of-requires :-)